### ISSUE TITLE
user build from IBM Z open editor gets stuck at 92% even after a successful build

### ISSUE THREAD

**Issue by Madhubalan-p** [IBM/zopeneditor-about#446]
## Development environment used

- Z Open Editor version: 4.5.1
- Editor Platform
  - [ ] Visual Studio Code
- Editor Platform Version: 1.92.2 (system setup)
- Operating System (on which VS Code runs such as Windows 10 2004, otherwise name and version of platform such as OpenShift v4.3): Windows 10 - Windows_NT x64 10.0.19045
- Java Version (when using VS Code or Theia, execute `java -version` and paste the details here): openjdk version "21.0.3" 2024-04-16 LTS
- [ ] Related to RSE API?
  - RSE API Plugin version: 1.1.0
  - Zowe CLI version: 7.29.2
  - Node.js version: 20.15.1
- Logs attached (see [here](https://ibm.github.io/zopeneditor-about/Docs/locating_local_client_logs.html) how to get them): no

## Problem Description

Detailed steps for reproducing the problem:
I have my zapp.yaml in my repo configured to run userbuild which looks like below.
![image](https://github.com/user-attachments/assets/113759b7-29b7-47b1-9314-b82d384cf61f)

when i run a userbuild from a cobol program, almost everything gets completed successfully. but the progress gets stuck at 92% after downloading log files locally. and cancelling the task doesn't help either. 
![image](https://github.com/user-attachments/assets/74319b16-b419-4d86-9512-bcf3c870b418)

there is nothing signifacnt in .zowe/zowe/logs
\AppData\Local\Temp\_ws_1727725077135\.metadata\.logs
\AppData\Roaming\Code\logs\20240930T223928\window1\exthost\IBM.zopeneditor

is this a known issue? what is the workaround for this? we really like the vs code for mainframe coding. so would be great if we could get around this issue, or solve it even better.

**Comment by phaumer** [comment]
Thanks. We are investigating the problem.

**Comment by Madhubalan-p** [comment]
> Thanks. We are investigating the problem.

Thanks for acknowledging it immediately. eagerly waiting for a solution.

**Comment by Madhubalan-p** [comment]
Hi @phaumer , any update on this issue? Funny thing was that this problem disappeared for a while then reappeared now. It gets stuck after 92% saying that **Running user build: Handling problems view (92%)**
i dont if it would be resolved once we update to version 5.0.0 and zowe 3.0.0.

**Comment by phaumer** [comment]
Yep, fixed it myself, actually. It will be in the release announced for Thursday.

**Comment by Madhubalan-p** [comment]
@phaumer  
awesome. can't wait to try. 🥳🥳🥳

**Comment by phaumer** [comment]
Fixed in 5.0.0. Let us know if it worked for you as well.
